### PR TITLE
[stable16] New Password Welcome form : Activate and choose intial password for new provisioned user

### PR DESCRIPTION
**Pull request by compagnon**
Proposal for #16245 
Committer: same person - @gcompagnon @compagnon

/newpassword 
/newpassword/email 
/newpassword/userid
would provide a dedicated welcome page for new users , letting them enter the same process than Lost Password (new password feature is implemented into LostController)
This URL could be used into a customized Welcome email in a simple way for Business people/sales.

The new user could use that URL to choose his password, and the form could be customized to make user to accept conditions of use.

image
If the user has never login(is enabled) => this is a First connection 
image

If the user returns, on that URL: this is a Lost Password form
image

* I have added also an administrator_email property on the config.php to display 'mailto' link to the * administrator email with the user id. l have updated some i10n messages for using that info instead of a "Administrator" , user knows the email. 
* Anti spam is enabled with throttle_delay

Here is an exception management example: a mailto on the administrator email is enabled
image

### CODE REVIEW

**Comment by kesselb** [comment]
Thank you for this contribution :tada: 

Mind to rebase the branch against master? New features are developed against the master branch.

File: .gitignore [hunk 1]
```diff
@@ -151,3 +151,6 @@ clover.xml
 # Tests - dependencies
 tests/acceptance/composer.lock
 tests/acceptance/vendor/
+config/config.php.test
```
Comment (kesselb) [inline .gitignore]: Please revert this

File: .htaccess [hunk 2]
```diff
@@ -65,3 +65,7 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
```
Comment (kesselb) [inline .htaccess]: Please revert this

File: tests/nextcloudPostgresql.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+SELECT table_name FROM information_schema.tables WHERE table_schema='public';
+
```
Comment (kesselb) [inline tests/nextcloudPostgresql.sql]: Please remove this file

File: .gitignore (hunk 1 above)
Comment (compagnon) [reply .gitignore]: #16257  is the  pullrequest on MASTER wo working files

**Comment by compagnon** [comment]
#16260 is the new pull request on Master

**Pull request closed without merging**